satflow/models/base.py

```diff
@@ -4,9 +4,49 @@
 import torchvision
 from neptune.new.types import File
 from satflow.models.hub import load_model_config_from_hf, load_pretrained
+from huggingface_hub import PyTorchModelHubMixin, PYTORCH_WEIGHTS_NAME
+import os
 
 REGISTERED_MODELS = {}
 
+MODEL_CARD_MARKDOWN = """---
+license: mit
+tags:
+- satflow
```

Comment:
@jacobbieker I picked a minimal set of tags (we use them for filtering / defining the interface for the inference API), but we can add whatever tags you think would be a useful descriptor of `satflow` models

Reply:
Okay! I'll think about ones to add, the ones I can think of so far would be something like `forecasting`, `remote-sensing`, or `timeseries`